go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ONSdigital/dp-frontend-search-controller
 
-go 1.19
+go 1.21
```

Comment:
You've set the language version 1.21, but are using a 1.20 docker image. This isn't a problem if the docker image is upgraded.

Reply:
I had set the app to use `1.21` and `1.21.1` but there missed this one is downgrading the version. Will push an update

Reply:
Set to `1.21`